react/features/shared-video/middleware.any.js

```diff
@@ -51,7 +52,13 @@ MiddlewareRegistry.register(store => next => action => {
                 if (isSharingStatus(sharedVideoStatus)) {
                     handleSharingVideoStatus(store, value, attributes, conference);
                 } else if (sharedVideoStatus === 'stop') {
-                    dispatch(participantLeft(value, conference));
+                    const videoParticipant = getParticipantById(state, value);
+
+                    dispatch(participantLeft(value, conference, {
+                        isFakeParticipant: videoParticipant?.isFakeParticipant,
+                        isVirtualScreenshareParticipant: videoParticipant?.isFakeParticipant
```

Comment:
Is `isVirtualScreenshareParticipant` needed here?

Reply:
I rechecked the flow with SHARED_VIDEO, no we don't need as virtual screenshare participant is on sharing the screen, for share video we only need isFakeParticipant